contracts/interfaces/AmunBasket.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: MIT
+pragma experimental ABIEncoderV2;
+pragma solidity ^0.7.1;
+
+interface IAmunBasket {
```

Comment:
There are owner only functions here. 

I believe it would be good only add the interface functions that zerion adapters interface to

Reply:
ok